HOW_TO_RUN.md

```diff
@@ -1,52 +1,60 @@
-Thi
+# starbake
 
-## Install 
+## Prerequisites
+
+Before installing starbake, ensure the following minimum versions are installed on your system:
+
+- jdk: 11 or higher
+- python: 3.8 or higher
+
+## Install
 
 1. Install Starlake
-    `SL_VERSION=1.0.0-SNAPSHOT sh /Users/elarib/Work/Starlake/starlake/distrib/starlake.sh`
-2. Generate dummy files
-   `pip install faker`
-   `python _scripts/dummy_data_generator.py`
+`sh <(curl https://raw.githubusercontent.com/starlake-ai/starlake/master/distrib/setup.sh) --target=.`
+2. Create a virtual environment (optional)
+   `python3 -m pip install virtualenv`
+   `python3 -m venv .venv`
+3. Activate the virtual environment (optional)
+   `source .venv/bin/activate`
+4. Generate dummy files
+   `python3 -m pip install -r _scripts/requirements.txt`
+   `python3 _scripts/dummy_data_generator.py`
 
 We're good to go
 
-
-## Run Starlake 
+## Run Starlake
 
 1. Import from incoming to pending
-`sh starbake.sh import`
-
+`./starlake import`
 
 2. Load data to bigquery
-`sh starbake.sh load`
+`./starlake load`
 
 3. Run the transformation in order
-```
-sh starbake.sh transform --name Customers.CustomerLifetimeValue 
-sh starbake.sh transform --name Customers.HighValueCustomers 
-
-sh starbake.sh transform --name Products.ProductProfitability 
-sh starbake.sh transform --name Products.MostProfitableProducts 
 
-sh starbake.sh transform --name Products.ProductPerformance 
-
-sh starbake.sh transform --name Products.TopSellingProducts 
-sh starbake.sh transform --name Products.TopSellingProfitableProducts 
-```
+```bash
+./starlake transform --name Customers.CustomerLifetimeValue 
+./starlake transform --name Customers.HighValueCustomers 
 
+./starlake transform --name Products.ProductProfitability 
+./starlake transform --name Products.MostProfitableProducts 
 
+./starlake transform --name Products.ProductPerformance 
 
+./starlake transform --name Products.TopSellingProducts 
+./starlake transform --name Products.TopSellingProfitableProducts 
+```
 
+## Run DAGs
 
+1. Install the dagster webserver
+   `python3 -m pip install dagster-webserver`
+2. Install the starlake dagster libraries for shell
+   `python3 -m pip install starlake-dagster[shell]`
```

Comment:
Can we change this line to `python3 -m pip install 'starlake-dagster[shell]'`
I have zsh and brackets should be quoted
![image](https://github.com/starlake-ai/StarBake/assets/6231666/6b097389-fb24-4c87-ad53-5044d9e7f420)